src/App.js

```diff
@@ -1,5 +1,37 @@
+import { MissionUtils } from '@woowacourse/mission-utils'
+import { MESSAGE } from './constants/message.js'
+import {
+	inputCarsNameAndValidate,
+	inputTryCountAndValidate,
+	printResult,
+	racingResult,
+	winner,
+} from './utils/index.js'
+
 class App {
-  async play() {}
+	async play() {
+		//자동차 이름 입력
+		MissionUtils.Console.print(MESSAGE.START)
+		const carsNameSplit = await inputCarsNameAndValidate()
```

Comment:
```suggestion
		MissionUtils.Console.print(MESSAGE.START)
		const carsNameSplit = await inputCarsNameAndValidate()
```
개인적으로는 따로 프린트 함수를 호출하는 것보다 `MissionUtils.Console.readLineAsync` 함수에 인자를 활용해서 출력해도 좋을 것 같아요!

Reply:
저도 동의합니다! 😄 
`inputCarsNameAndValidate()`함수 내에서 `MissionUtils.Console.readLineAsync("")` 대신 `MissionUtils.Console.readLineAsync(MESSAGE.START)` 이렇게 수정하시면 `MissionUtils.Console.print(MESSAGE.START)`는 제거할 수 있어서 더 간결해질 것 같습니당

Reply:
오~ 이 방법은 생각하지 못했네요! 변경해야겠군요!